en/ios/user-interface.mdown

```diff
@@ -484,8 +484,56 @@ The easiest way to understand this class is with an example. This subclass of `P
 @end
 ```
 ```swift
-// no Swift example
-// Want to contribute to this doc? https://github.com/ParsePlatform/Docs
+class SimpleTableViewController : PFQueryTableViewController {
+    
+    override init(style: UITableViewStyle, className: String?) {
+        super.init(style: style, className: className)
+        parseClassName = "Todo"
+        pullToRefreshEnabled = true
+        paginationEnabled = true
+        objectsPerPage = 25
+    }
+
+    required init?(coder aDecoder: NSCoder!) {
```

Comment:
Looks like you missed one nit. `aDecoder` here, according to the latest Swift 2 docs is never `nil`, meaning that you can skip implicitly unwrapped optional aka remove exclamation mark.